src/value.cpp

```diff
@@ -1013,8 +1013,8 @@ std::ostream& TimeValue::write(std::ostream& os) const {
 
 int64_t TimeValue::toInt64(size_t /*n*/) const {
   // Returns number of seconds in the day in UTC.
-  int64_t result = (time_.hour - time_.tzHour) * 60 * 60;
-  result += (time_.minute - time_.tzMinute) * 60;
+  auto result = int64_t(time_.hour - time_.tzHour) * 60 * 60;
```

Comment:
I do not see any issue in the previous code. All the members in the `Time` struct are `int`s and we have enough precision there to not have issues with casting.

Furthermore you are starting to use the C-Style casts here, and I have a strong preference for the usage of C++ casts. 

I also think that the change in `xmpsidecar.cpp` is not necessary.

Reply:
Right. The particular warning is about implicit widening.